CharacterCreation/01_02_Species_ECSC_Security_Drone.txt

```diff
@@ -229,3 +229,11 @@ drones, to deal with the constant heavyweight lifting that is required of them.
     
 Size: Large
 Favored Terrain: Urban
+
+Engineering-Minded:
+    The Series Twelve Drone is programmed to better understand structural stability and 
+weaknesses. Knowledge is powerful, and Constructor Series Drones know exactly where to 
+apply force to do the most damage. Gain +20 when trying to weaken or demolish structures 
+such as walls or doors. Constructor Drones are built to be very resourceful when it comes
+to building. Gain Jury-Rigging at 15 if you take it as a level 1 skill, or at 20 if you 
```

Comment:
15 and 20 are a little too high for level 1 with @rosebrallier's new skill system, where 6 or 12 would be the max expected of a level 1 character. Also, Engineers already get a flat bonus of 10 to Jury-rigging as well as proficiency in the skill. I really like the idea that construction bots are better at learning to build things, so I hope we keep the theme. I really like it. Rev suggests changing this to advantage on jury-rigging instead?